packages/core/src/lib/implementation/persist.ts

```diff
@@ -22,15 +21,13 @@ export class PersistError extends Error {
   }
 }
 
-export type PersistResult = PromiseSettledResult<readonly [string, number]>[];
-
 export async function persistReport(
   report: Report,
   config: CoreConfig,
-): Promise<PersistResult> {
+): Promise<MultipleFileResults> {
```

Comment:
This is still handling multiple files. Please revert.

Reply:
It can save `json`, `md`, etc... And the new type is just more generic. So i did not change the logic.

Please help me to understand what exactly i should do.

Reply:
Oh, I think I got confused :man_facepalming: Thanks for explaining, multiple formats of the same report is fine, of course :+1:

You can ignore my comment here, I mixed this up with `loadReports` which is scanning the output directory for multiple different reports, that's what should be reverted. But I already mentioned that here: https://github.com/flowup/quality-metrics-cli/pull/187#discussion_r1383153436